models/Tool/CustomReport/Adapter/Sql.php

```diff
@@ -38,7 +38,8 @@ public function getData($filters, $sort, $dir, $offset, $limit, $fields = null,
 
             $order = '';
             if ($sort && $dir) {
-                $order = ' ORDER BY ' . $db->quoteIdentifier($sort) . ' ' . $dir;
+                $dir = ((strtolower($dir) == 'ASC') ? 'ASC' : 'DESC');
```

Comment:
You mean `strtoupper()`, right?

```suggestion
                $dir = ((strtoupper($dir) === 'ASC') ? 'ASC' : 'DESC');
```

Reply:
yes, exactly - thanks :)